earth_enterprise/rpms/opengee-common/pre-install.sh

```diff
@@ -24,9 +24,6 @@ check_group()
     # Add group if it does not exist:
     if [ -z "$GROUP_EXISTS" ]; then
```

Comment:
Suggestion: You can replace this function with `groupadd -f -r "$GEGROUP"`.

Reply:
is it valid for both rhel6 and 7?

Reply:
Yes.